flake.nix

```diff
@@ -0,0 +1,95 @@
+{
+  description =
+    "Decentralized search engine & automatized press reviews - firefox extension";
+
+  # Nixpkgs / NixOS version to use.
+  inputs.nixpkgs.url = "nixpkgs/21.05";
+
+  inputs.meta-press = {
```

Comment:
Name this `meta-press-src`. This avoids any aliasing between the package and the derivation that shares the same name.